src/components/Blinds/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+//import 'style.css';
```

Comment:
Hele tohle by normálně mělo fungovat, co jsi tam konkrétně měla za problém že to nešlo?